### ISSUE TITLE
[BUG] Target ID overlaps elements when in minmode

### ISSUE THREAD

**Issue by impale1** [CriticalFlaw/TF2HUD.Fixes#136]
https://github.com/ValveSoftware/Source-1-Games/issues/4905
**Issue**
I was wrong, I'm sorry.
The problem was not caused by changed position of the target id, but I will say that since "default" is now at the bottom like "bottom center" it will confuse players

This setting is not hidden somewhere in the depths of the console and known only to experienced users, it can be easily found in the advanced options
![1](https://user-images.githubusercontent.com/117596825/231861001-2c712170-bbb2-4e93-a35d-ca36c88fb1bd.png)
`tf_spectator_target_location` (number)
0 - default
1 - bottom left corner
2 - bottom center
3 - bottom right corner

**Specifications**
Operating system - Fedora Linux 37 (Workstation Edition)
Resolution - 1920x1080

**Screenshots**
`tf_spectator_target_location 2` is used
![hudfixes2](https://user-images.githubusercontent.com/117596825/231718279-6b044433-f13f-482d-9472-a7c84fe6d58c.png)

**Comment by CriticalFlaw** [comment]
afaik the target ID position values when changed through the advanced options menu are set in the game code, not the HUD files. In which case it would have to be adjusted by Valve themselves. Going through the HUD files I was not able to find anything relevant to this feature.

**Comment by mtxfellen** [comment]
By maintaining the ypos values of `bottombarblank` in spectator.res and instead changing the xpos value like so, you can achieve the desired effect:
```
	"bottombarblank"
	{
		"ControlName"	"Panel"
		"fieldName"		"bottombarblank"
		"xpos"			"9999"
		"ypos"			"r70"	[$WIN32]
		"ypos"			"r90"	[$X360]		
		"ypos_minmode"	"r57"
		"tall"			"70"	[$WIN32]		// this needs to match the size of BottomBar
		"tall"			"90"	[$X360]		
		"wide"			"f0"
		"autoResize"	"0"
		"pinCorner"		"0"
		"visible"		"0"
		"enabled"		"0"
		"tabPosition"	"0"
	}
```
Here's how it looks:
Minmode off:
![](https://i.imgur.com/y4rmodh.png)
Minmode on:
![](https://i.imgur.com/SJDYmTP.png)

**Comment by CriticalFlaw** [comment]
I don't understand the benefit of that feature to begin with. Moving it to either side will cause to overlap with so much stuff while in spectator.

**Comment by CriticalFlaw** [comment]
Fixed in 9f2652f0d612af1f7da9d5450e410f3cf0805041. Thanks to @mtxfellen
